release/tasks/disk-virt-customize/disk-virt-customize.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
     tekton.dev/categories: Automation
     tekton.dev/tags: kubevirt
     tekton.dev/displayName: "KubeVirt disk virt customize"
-    tekton.dev/platforms: "linux/amd64"
+    tekton.dev/platforms: "linux/amd64,linux/arm64"
```

Comment:
We need `quay.io/kubevirt/libguestfs-tools` for `s390x` to add it here, correct?

Reply:
Hi @0xFelix currently we dont have s390x image of `quay.io/kubevirt/libguestfs-tools` , and we are working on it to have s390x multiarch image. once the "quay.io/kubevirt/libguestfs-tools" have multi arch s390x image, i will add it in separate PR.

Reply:
Sure!